src/main/resources/config/liquibase/changelog/20240729103000_changelog.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240729103000" author="janthoXO">
+        <createTable tableName="ide">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="idePK"/>
+            </column>
+            <column name="name" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="deep_link" type="varchar(200)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+        </createTable>
+        <createTable tableName="user_ide_mapping">
+            <column name="user_id" type="bigint">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="user_ide_mappingPK"/>
+            </column>
+            <column name="programming_language" type="varchar(50)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="user_ide_mappingPK"/>
+            </column>
```

Comment:
maybe you could use a simple enum type (e.g. tinyint) here, as that would save a lot of space. We probably won't have more than 255 programming languages anyway :)

Reply:
i've implemented it with a simple enum type, but according to @bassner we use it like this in artemis